EIPS/eip-6366.md

```diff
@@ -33,113 +34,26 @@ Compliant contracts MUST implement `IEIP6366Core`.
-```
-
-The most important permission SHOULD be greater than the lesser one.
-
-#### transfer
-
-Transfers a subset of `_permission` permission to address `_to`, and MUST emit the `Transfer` event.
-The function SHOULD `revert` if the message caller's account permission does not have the subset of the transferring permission. The function SHOULD `revert` if any of transferring permission is existing on target `_to` address.
-
-_Note_ Transfers of `0` permission MUST be treated as normal transfers and emit the `Transfer` event.
-
-```solidity
-function transfer(address _to, uint256 _permission) external returns (bool success);
-```
-
-#### approve
-
-Allows `_delegatee` to act for the permission owner's behalf, up to the `_permission` permission, and MUST emit the `Approval` event. If this function is called again it overwrites the current granted with `_permission`.
-
-_Note_ `_permission` MUST be a subset of all available permission of permission owner.
-
-_Note_ `approve()` method SHOULD `revert` if granting `_permission` permission is not a subset of all available permission of permission owner.
-
-```solidity
-function approve(address _delegatee, uint256 _permission) external returns (bool success);
-```
-
-#### permissionOf
-
-Returns the account permission of the given `_owner` address.
-
-```solidity
-function permissionOf(address _owner) external view returns (uint256 permission);
-```
-
-#### permissionRequire
-
-Returns `true` if `_required` permission is a subset of `_permission` permission otherwise return `false`.
-
-```solidity
-function permissionRequire(uint256 _required, uint256 _permission) external view returns (bool isPermissioned);
-```
-
-### hasPermission
-
-Returns `true` if `_required` permission is a subset of `_actor`'s permissions or a subset of his delegated permissions granted by the `_owner`.
-
-```solidity
-function hasPermission(address _owner, address _actor, uint256 _required) external view returns (bool isPermissioned);
-```
-
-#### delegated
-
-Returns the subset permission of the `_owner` address were granted to `_delegatee` address.
-
-```solidity
-function delegated(address _owner, address _delegatee) external view returns (uint256 permission);
-```
-
-#### Transfer
-
-MUST trigger when permission are transferred, including zero permission transfers.
-
-A token contract which creates new tokens SHOULD emit a `Transfer` event with the `_from` address set to `address(0x00)` when tokens are created.
-
-```solidity
-event Transfer(address indexed _from, address indexed _to, uint256 _value)
-```
-
-#### Approval
-
-MUST trigger on any successful call to `approve(address _delegatee, uint256 _permission)`.
-
-```solidity
-event Approval(address indexed _owner, address indexed _delegatee, uint256 _permission)
-```
+1. It is RECOMMENDED to define each permission as a power of `2` so that we can check for the relationship between sets of permissions using [eip-bit_based_permission](./bit_based_permission.md).
```

Comment:
It feels to use `RECOMMENDED` here and `MUST` in the bit-based eip [here](https://github.com/orochi-network/EIPs/blob/a8937573b2a289286ff3cc51c87fb615e2a2af23/EIPS/eip-bit_based_permission.md?plain=1#L44)

Reply:
This proposal have different purpose and we allow user to define it themselves but the best practice is Bit Based Permission.